### DISCUSSION TITLE
Authentication Captcha For Organization Creation

### DISCUSSION THREAD

**Discussion by doubleprincez** [community/community#47558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Unable to verify your captcha answer. Please try again or visit https://docs.github.com/articles/troubleshooting-connectivity-problems/#troubleshooting-the-captcha for troubleshooting information.
I have been trying to fix this challenge for more than 3 years now and I am currently using a new chrome on a new PC but still the same issue.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @doubleprincez ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by EranBoudjnah** [comment]
This is an issue reported in 2023. I'm facing the exact same issue today. There's no captcha challenge presented - only a Github spinner followed by a green checkmark. Yet the creation fails (with no user facing error) until I navigate away. At that point, I see the same error as the original reporter mentioned.

**Reply by EranBoudjnah** [reply]
Just a quick update: two browsers (Chrome and Vivaldi) failed, but Vivaldi from my Android phone worked.
